.github/workflows/build.yml

```diff
@@ -388,13 +409,20 @@ jobs:
           name: Oculus Quest Experimental
           path: build_oculus_quest_experimental
 
+      - name: Download Build Artifacts (Pico)
```

Comment:
One last nitpick; if you build pico experimental, it should be added to the release here and in the github release creation. Other than that, looks great!